### PR TITLE
option to display unit vector

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
Just a small option to adapt format of vector field.

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...).
- [x] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -6,7 +6,10 @@
     (Bertrand Kerautret, [#296](https://github.com/DGtal-team/DGtalTools/pull/296))
   - meshViewer: add an option to set the ambient light source.
     (Bertrand Kerautret, [#301](https://github.com/DGtal-team/DGtalTools/pull/301))
-
+  - 3dSDPViewer: new option to display vector field as unit vector
```
Comment (dcoeurjo) [inline ChangeLog.md]: as unit vectors ?

File: visualisation/3dSDPViewer.cpp [hunk 2]
```diff
@@ -433,9 +444,16 @@ int main( int argc, char** argv )
           double percentage = vm["filterVectors"].as<double>();
           step = max(1, (int) (100/percentage));
         }
-      for(unsigned int i =0; i<vectorsPt.size()-1; i=i+2*step)
-      {
-        viewer.addLine(vectorsPt.at(i),vectorsPt.at(i+1), lineSize);
+      if(useUnitVector){
```
Comment (dcoeurjo) [inline visualisation/3dSDPViewer.cpp]: coding style ?

:)

File: ChangeLog.md (hunk 1 above)
Comment (kerautret) [inline ChangeLog.md]: yes

**Comment by kerautret** [comment]
done thanks ;)

**Comment by dcoeurjo** [comment]
thanks merging

**Pull request merged**